sdk/python/lib/pulumi/automation/_local_workspace.py

```diff
@@ -299,16 +299,20 @@ def list_stacks(self) -> List[StackSummary]:
             stack_list.append(stack)
         return stack_list
 
-    def install_plugin(self, name: str, version: str, kind: str = "resource") -> None:
-        self._run_pulumi_cmd_sync(["plugin", "install", kind, name, version])
+    def install_plugin(self, name: str, version: str) -> None:
+        self._run_pulumi_cmd_sync(["plugin", "install", "resource", name, version])
+
+    def install_plugin_from_server(self, name: str, version: str, server: str) -> None:
```

Comment:
Similarly, I think we'll want to add `install_plugin_from_server` as an abstract method of the `Workspace` class in `_workspace.py`.